### PR TITLE
User can create recipe

### PR DESCRIPTION
**Pull request by koggs**
#### What I did:
- Add form for recipes
- Add routes
- Generate recipe controller 
- Add links to show and create

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -20,6 +20,8 @@ group :development, :test do
   gem 'rspec-rails'
   gem 'shoulda-matchers'
   gem 'simplecov', require: false, group: :test
+  gem 'pry-byebug'
+  gem 'pry-byebug'
```
Comment (juliaramstedt) [inline Gemfile]: twice :)

File: app/controllers/recipes_controller.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      redirect_to recipe_path(@recipe)
+    else
+      flash[:error] = "Something went wrong"
+      render 'new'
+    end
+  end
+  
+  
+  private
+  def recipe_params
```
Comment (juliaramstedt) [inline app/controllers/recipes_controller.rb:29]: Add a line break after "private". Rubocop should warn about this... is rubocop set up? You can test to run rubocop from the command line, just type  `rubocop`.

Comment (koggs) [reply app/controllers/recipes_controller.rb:29]: it is now :)

File: Gemfile (hunk 1 above)
Comment (koggs) [reply Gemfile]: done

**Pull request merged**